include/deal.II/base/geometry_info.h

```diff
@@ -2132,6 +2203,18 @@ struct GeometryInfo
     vertex_to_face =
       internal::GeometryInfoHelper::Initializers<dim>::vertex_to_face();
 
+  /**
+   * This field stores for each line to which faces it belongs for a 3d
+   * reference cell. The first index in this 2d-array runs over all lines, the
+   * second index over two faces to which the line belongs.
+   *
+   * The order of the faces for each line is such that the first listed face
+   * bounds the reference cell in <i>x</i> direction, the second in <i>y</i>
+   * direction, and so on.
+   */
+  static constexpr ndarray<unsigned int, 12, 2> line_to_face =
+    internal::GeometryInfoHelper::Initializers<3>::line_to_face();
```

Comment:
I am not really sure, if it is really good from a conceptual point of view to include this newly introduced function which is only valid for a 3D reference cell here. I could also shift it to Internal somewhere.